enclave-tls/src/enclave_quotes/sgx-ecdsa/verify_evidence.c

```diff
@@ -42,6 +42,12 @@ enclave_quote_err_t sgx_ecdsa_verify_evidence(enclave_quote_ctx_t *ctx,
 	ETLS_DEBUG("quote size is %d, quote signature_data_len is %d\n",
 		   quote_size, pquote->signature_data_len);
 
+	// First verify the hash value
+	if (*hash != (uint8_t *)pquote->report_body.report_data.d) {
```

Comment:
The hash is a fixed-length buffer so it should compare the buffer rather than the first byte.

Reply:
Changed. Please help review again.